index.html

```diff
@@ -0,0 +1,643 @@
+                    <div class="img-text">
+                    <small>MOST POPULAR</small> 
+                    <h5>How to manage your screen time during the lockdown, according to science</h5>
+                    <small>The Conversation day 4</small>
+                      </div>
+                    <div class="blog-card-overlay"></div>
+                </div>
+                    <div class="inner-img">
+                        <p><span>LATEST</span> <br> How sleep trackers can actually ruin your sleep</p> 
+                        <img src="assets/images/20.jpg" alt="">
+                    </div>
+                <div class="card-links">
+                    <ul>
+                        <li><a href="#">Report: Apple is building foldable iPhone prototypes</a></li>
+                        <li><a href="#">The Galaxy Buds Pro are Samsung's answer to the AirPods Pro</a></li>
+                        <li><a href="">Samsung reveals the Galaxy S21 series, $200 cheaper this time</a></li>
+                        <li><a href="">HTC is apparently still making phones, announces the Desire 21 Pro 5G</a></li>
+                        <li><a href="">LG confirms its shape-shifting "rollable" phone arrives this year</a></li>
+                    </ul>
+                </div>
+            </div>
+
+            <div class="blogs-card">
+               <a href="#" class="top-big-link"><h3>Tech ></h3></a> 
+                <div class="main-img fouth"> 
+                    <div class="img-text">
+                    <small>MOST POPULAR</small>
+                    <h5>You won't need wing mirrors in the US any more — if you're a self-driving car</h5>
+                    <small>Matthew Beedham11 hours ago</small>
+                </div>
+                <div class="blog-card-overlay"></div>
+                      </div>
+                    <div class="inner-img">
+                        <p><span>LATEST</span> <br> Rolls-Royce boldly skips hybrids and goes all-electric with new EV lineup</p> 
+                        <img src="assets/images/6.jpg" alt="">
+                    </div>
+                <div class="card-links">
+                    <ul>
+                        <li><a href="">What is semi-supervised machine learning?</a></li>
+                        <li><a href="">You won't need wing mirrors in the US any more — if you're a self-driving car</a></li>
+                        <li><a href="">Forget coding, you can now solve your AI problems with Excel</a></li>
+                        <li><a href="">Scientists figured out how to turn black holes into power sources</a></li>
+                        <li><a href="">How to manipulate date and time in JavaScript</a></li>
+                    </ul>
+                </div>
+            </div>
+
+            <div class="blogs-card">
+                <a href="#" class="top-big-link"><h3>Podium ></h3></a>
+                <div class="main-img fifth" style="background-image: url(assets/images/21.jpg);">
```

Comment:
- [ ] remove inline stylings  ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/html_css/section/css_bad_practices_requirement.png) from Pathwright) ([article mentioned in the screenshot](https://speckyboy.com/good-bad-css-practices))